### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by marcellorinaldo**
Signed-off-by: Marcello Martina <martina.marcello.rinaldo@outlook.com>

Brief description of the PR. Documentation updates

**Related Issue:** The actual documentation was not up-to-date. Some steps were wrong and some images didn't correspond to the newest version of kura.

**Description of the solution adopted:** Walkthrough of the documentation to find errors. Updates made to the GitHub pages markdown files.

**Screenshots:** N/A.

**Any side note on the changes made:** N/A.

### CODE REVIEW

File: _posts/config/2017-05-04-device-information.md [hunk 1]
```diff
@@ -21,10 +21,12 @@ The **Threads** tab shows a list of the threads that are currently running in th
 
 ![]({{ site.baseurl }}/assets/images/config/DeviceThreads.png)
 
+<!---
 ## System Packages
 The **System Packages** tab shows the list of all the Linux packages installed on the OS. The package is detailed with the name, version and type (DEB/RPM).
 
 ![]({{ site.baseurl }}/assets/images/config/DeviceSystemPackages.png)
+--->
```
Comment (pierantoniomerlino) [inline _posts/config/2017-05-04-device-information.md]: This one is for a Kura 5 feature.

File: _posts/config/2017-05-04-device-information.md [hunk 2]
```diff
@@ -34,7 +36,10 @@ The **System Properties** tab shows a list of relevant properties including OS a
...
 
+<!---
 ## System Logs
 The **System Logs** tab allows to download a compressed file containing all the relevant log files from the gateway.
 
-![]({{ site.baseurl }}/assets/images/config/DeviceSystemLogs.png)
\ No newline at end of file
+![]({{ site.baseurl }}/assets/images/config/DeviceSystemLogs.png)
+
+--->
```
Comment (pierantoniomerlino) [inline _posts/config/2017-05-04-device-information.md]: This one is for a Kura 5 feature.

File: _posts/config/2017-06-10-threat-manager.md [hunk 3]
```diff
@@ -1,44 +0,0 @@
...
-iptables -A prerouting-kura -p tcp --tcp-flags ALL SYN,FIN,PSH,URG -j DROP
-iptables -A prerouting-kura -p tcp --tcp-flags ALL SYN,RST,ACK,FIN,URG -j DROP
-iptables -A prerouting-kura -p icmp -j DROP
-iptables -A prerouting-kura -f -j DROP
-```
-
-The following parameters are available:
-
-- *flooding.protection.enabled* : enables the application of the firewall rules for flooding protection.
-
```
Comment (pierantoniomerlino) [inline _posts/config/2017-06-10-threat-manager.md]: This one is for a Kura 5 feature.

File: _posts/intro/2014-02-01-raspberry-pi-quick-start.md [hunk 4]
```diff
@@ -91,16 +91,18 @@ following steps:
...
-     ![Untrusted certificate details]({{ site.baseurl }}/assets/images/admin/untrusted_cert2.png)
+     <!---![Untrusted certificate details]({{ site.baseurl }}/assets/images/admin/untrusted_cert2.png)--->
 
-     ![Proceed trusting the source]({{ site.baseurl }}/assets/images/admin/untrusted_cert3.png)
+     <!---![Proceed trusting the source]({{ site.baseurl }}/assets/images/admin/untrusted_cert3.png)--->
 
-     Once trusted the source, the user will be redirected to a login page where the default **username** is:
+     <!---Once trusted the source, the user will be redirected to a login page where the default **username** is:--->
+     
+     The user will be prompted for login credentials where the default **username** is:
```
Comment (pierantoniomerlino) [inline _posts/intro/2014-02-01-raspberry-pi-quick-start.md]: This one is for a Kura 5 feature.

**Review by pierantoniomerlino — APPROVED** [review]

**Comment by MMaiero** [comment]
Since we don't have the capability to separate the docs per version, maybe we can add a note on what is for Kura 5 and what is for Kura 4.

**Pull request merged**